### ISSUE TITLE
a11y - Axe core gems

### ISSUE THREAD

**Issue by ewlarson** [UMNLibraries/cdm-blacklightify#25]
### Descriptive summary

We should add the [axe-core-gems](https://github.com/dequelabs/axe-core-gems) to the project and perhaps even wire them up to our CI actions.
